test/utils.py

```diff
@@ -234,21 +249,45 @@ def start(self, connection_test=True,
                 instance within three seconds deadline after port bound (an attempt to connect is
                 made if there is an option connection_test=True that is present by default).
         """
-        popen_obj = subprocess.Popen(["tarantool", self._instance_cfg_file_name], cwd=self._tmpdir)
-        file_with_port_path = str(self._tmpdir) + '/' + self._instance_cfg_file_name + '.port'
-
-        # Waiting 3 seconds for instance configure itself and dump bound port to temp file.
-        deadline = time.time() + 3
-        while time.time() < deadline:
-            if os.path.exists(file_with_port_path) and os.path.getsize(file_with_port_path) > 0:
-                break
-            time.sleep(0.1)
+        tnt_start_cmd = []
+        major_version, _ = get_tarantool_version()
+        if major_version < 3 or use_lua:
+            if instance_name != "" and (".yml" in self._instance_cfg_file_name or
+                                        ".yaml" in self._instance_cfg_file_name):
+                raise Exception("there is no instance_name for Tarantool version < 3")
```

Comment:
The error is a little bit confusing.
```suggestion
                raise Exception("instance_name cannot be used with Tarantool version < 3")
```

Reply:
Edited.